app/assets/stylesheets/common.scss

```
@@ -799,6 +799,15 @@ tr.turn:hover {
   }
 }
 
+/* Rules for notes pages */
+
+@include color-mode(dark) {
+  .notes .table-primary td {
```

```suggestion
  .notes .table-primary-subtle td {
```

And in https://github.com/openstreetmap/openstreetmap-website/pull/4693/files#diff-4fe647349351c7492508c788230b91b7f4445d4b821f6781ac7fcfad5e879f21R27 also `table-primary-subtle`.

---

There's no standard `table-primary-subtle` class. The background of `table-primary` happens to match `bg-primary-subtle` in light mode.